x/emissions/proto/emissions/v6/params.proto

```diff
@@ -156,5 +156,4 @@ message Params {
   // and participate in all topics as workers and reputers
   bool topic_creator_whitelist_enabled = 51; // topic creator whitelist enabled => only topic creator whitelisted actors can create topics
   uint64 min_experienced_worker_regrets = 52; // minimum number of experienced workers required to use their regrets
-  // for calculating the topic initial regret
```

Comment:
usually this file would not change and be frozen with a new migration...

Reply:
ty, it slipped in in my refactors. good catch, reverted it.